app/assets/src/components/visualizations/table/base_table.scss

```diff
@@ -134,3 +134,31 @@
     display: block;
   }
 }
+
+.selectableColumnClassName {
+  // LUCIA: We pass in the "selectableCell" className to
+  // our checkbox to modify the enclosing div as follows;
+  // this creates a large, clickable "button" the height of
+  // the row, making samples easier to select.
+  .selectableCell {
+    position: absolute;
+    overflow: visible;
+    height: 100%;
+    width: 50px;
+    margin: -11px -10px -10px -10px;
+    padding-left: 0px;
+    justify-content: center;
+    border-radius: 4px;
+
+    &:hover {
+      background-color: $primary-lighter;
+      opacity: 0.7;
+      transition: background-color 0.3s;
```

Comment:
We do not normally use transitions... I would not see a reason to start now... but again, wait on design input.